src/view.coffee

```diff
@@ -9,7 +9,7 @@ tree = require './tree'
 
 module.exports =
 class View extends Command
-  @commandNames: ['view', 'show']
+  @commandNames: ['view', 'show', 'v']
```

Comment:
Shouldn't `info` be added as an alias?

Homebrew uses it, and I'm constantly getting `info` mixed up with `show` or `view`...

Reply:
(more than a year later...)

Sounds good, included.